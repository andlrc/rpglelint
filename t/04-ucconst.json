[
   {
      "column":7,
      "file":"t/04-ucconst.rpgle",
      "line":2,
      "message":"constant 'c1' needs to be all uppercase",
      "severity":"warning"
   },
   {
      "column":13,
      "file":"t/04-ucconst.rpgle",
      "line":6,
      "message":"constant 'c3' needs to be all uppercase",
      "severity":"warning"
   }
]
