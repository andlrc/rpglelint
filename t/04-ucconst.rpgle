// rpglelint: -Wucconst
dcl-c c1 
const('123');
dcl-c C2 const('123');
dcl-proc my_proc;
  dcl-c     c3 const('123');
  dcl-c C4 const('123');
end-proc;
