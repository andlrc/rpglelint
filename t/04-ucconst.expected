t/04-ucconst.rpgle:2:7: warning: constant 'c1' needs to be all uppercase [-Wucconst]
 dcl-c c1 
       ^
t/04-ucconst.rpgle:6:13: warning: constant 'c3' needs to be all uppercase [-Wucconst]
   dcl-c     c3 const('123');
             ^
